Hazelcast.Net/Hazelcast.IO.Serialization/SerializationService.cs

```diff
@@ -682,11 +684,11 @@ private ISerializerAdapter SerializerFor(object obj)
             if (serializer == null)
             {
                 if (_isActive)
-                {
```

Comment:
please do not format or remove curly bracket from unrelated code like `SerializerFor` methods

Reply:
added missing brackets